src/include/atomic.h

```diff
@@ -55,7 +55,9 @@
 #ifndef PMIX_SYS_ATOMIC_H
 #define PMIX_SYS_ATOMIC_H 1
 
+#ifdef HAVE_STDATOMIC_H
```

Comment:
To avoid more header checks, I'd just put this include in the C11 block.  The C11 atomics test won't be true if stdatomic.h exists.  And if you're not using C11 atomics, this header's no use to you.  That would save yet another header test during configrue.

Reply:
Done